src/orgmode/inline.clj

```diff
@@ -40,25 +40,28 @@
 
 ;; ### Timestamp Elements
 
+;; hack to make it work with ranges
```

Comment:
Pedantic: Is it a _hack_ or a _fix_?

Reply:
I don't really know, honestly :-). It is hack in the sense that I don't why you get an intermediate grouping (my regexfu is limited), this is why it is hack. Empirically, it provides the requested results though.